Makefile

```diff
@@ -51,7 +50,7 @@ build:
 
 .PHONY: push
 push:
-ifeq ($(shell [[ $(BRANCH) != "master" && $(VERSION) != "dev" ]] && echo true ),true)
+ifeq ($(shell [[ $(BRANCH) != "release-0.2.0" && $(VERSION) != "dev" ]] && echo true ),true)
```

Comment:
Is this branch name change intentional?

Reply:
This was intentional, and is necessary for `make publish` to run on this branch. It's not a very good pre-release check to begin with, but this is a quick fix that unblocks publishing images for now. I've made a note to follow up con-1881 with a jira ticket to improve the pre-release checks on this repo (see description of #170)